vnext/Libraries/Text/TextProps.windows.js

```diff
@@ -0,0 +1,270 @@
+/**
```

Comment:
Are there corresponding typescript types to update? Like a Text.d.ts file or TextProps.ts file?

They might just live in DefinitelyTyped and you'll have to update them there. Might be a task for when this particular fix is upstreamed.

Reply:
There is a TextProps.js but I don't see any typescript files?